src/server.cpp

```diff
@@ -3,19 +3,15 @@
 #include <grpc/impl/codegen/byte_buffer_reader.h>
 #include "common.h"
 #include <string>
+#include <grpc/slice.h>
 
 using namespace Rcpp;
 
 #define _INTERRUPT_CHECK_PERIOD_MS 1000
 
 CharacterVector sliceToChar(grpc_slice slice){
 
-  char* data =
-  const_cast<char *>(reinterpret_cast<const char *>(
-      GRPC_SLICE_START_PTR(slice)));
-
-  // https://github.com/grpc/grpc/blob/master/src/core/lib/slice/slice.cc#L34
-  data[GRPC_SLICE_LENGTH(slice)] = 0;
```

Comment:
I wish i had commented why I did that, instead of just a link to the line that made it work :/